src/lib/utils/index.ts

```diff
@@ -8,25 +6,44 @@ interface DomainMetadataParams {
 	story: string;
 }
 
+interface UploadResponseInterface {
+	fleekHash: string;
+	hash: string;
+	url: string;
+}
+
+const apiEndpoint = `https://zns-backend.netlify.app/.netlify/functions/upload`;
+
 /**
  * Creates and uploads a domains metadata to IPFS
  * @param params Metadata parameters
  * @returns URI to the created Metadata
  */
+
 export const createDomainMetadata = async (params: DomainMetadataParams) => {
-	// upload image to IPFS
-	const { path: imageCid } = await ipfsClient.add(params.image);
+	// upload image to http backend
+
+	const imageResponse = await fetch(apiEndpoint, {
+		method: 'POST',
+		body: JSON.stringify(params),
```

Comment:
This is uploading the `DomianMetadataparams` when it should be uploading the image found in `params.image`